test/unit/org/apache/cassandra/io/sstable/SSTableUtils.java

```diff
@@ -162,7 +163,7 @@ public Context dest(Descriptor dest)
         /**
          * Sets the generation number for the generated SSTable. Ignored if "dest()" is set.
          */
-        public Context generation(int generation)
+        public Context generation(SSTableUniqueIdentifier generation)
```

Comment:
Nit: the JavaDoc for the method should say something like `Sets the identifier for the generated SSTable`